src/StatusQ/Components/StatusChatInfoToolBar.qml

```diff
@@ -28,6 +28,8 @@ Item {
         anchors.verticalCenter: parent.verticalCenter
         anchors.left: parent.left
         anchors.leftMargin: 5
+        anchors.right: statusMenuButton.left
+        anchors.rightMargin: 5
```

Comment:
There's a typo here so the margin is broken.

In addition:

1. This was "fixed" in https://github.com/status-im/StatusQ/commit/7ef61ed3f897d8b06c1185377de46dc173077edb but it looks like it requires the `width` of `StatusInfoButton` to be calculated. Here's how it was done in `StatusChatToolBar` which uses the same component: https://github.com/status-im/StatusQ/commit/7ef61ed3f897d8b06c1185377de46dc173077edb#diff-561ecc8f8bd7aa15fd3928afcad98931b5ceb20b570739485458be78dce8dd19R40
2. We should probably double check with designers if it's okay to have the button take the entire width here (which is what happens due to `anchors.right`). As per design, it should more behave like it was done in `StatusChatToolBar`.
2. If we go with this solution, we need to make sure this also works when no `popupMenu` is supplied. Because right now the button still bleeds out of the left panel
<img width="327" alt="Screenshot 2021-09-14 at 14 31 11" src="https://user-images.githubusercontent.com/445106/133257905-b542afb6-03d7-48ae-9b08-2532c9f15a46.png">

Reply:
@PascalPrecht not sure if I get it, this is what I see on my end:

With popup menu:
<img width="341" alt="withMenu" src="https://user-images.githubusercontent.com/31625338/133278689-8f00e911-e71b-4372-8d3a-093722424306.png">

Without popup menu:
<img width="340" alt="withoutMenu" src="https://user-images.githubusercontent.com/31625338/133278727-82c50173-3f29-4044-986c-014e1aa6dc86.png">

Reply:
Ah that's weird. I'm seeing the same as you do now. Probably forgot to reload the front-end.
This then looks good to me!

Just need to double check with design (@simonricoo are we good with having that button to take more space than needed? Designs usually show a width of button content + a tiny padding)

Reply:
@simonricoo specifically referring to this:

<img width="311" alt="Screenshot 2021-09-14 at 18 00 10" src="https://user-images.githubusercontent.com/445106/133292791-eab21024-0ff1-4241-858d-5dee92c4895a.png">

Reply:
> There's a typo here so the margin is broken.

Also this isn't true, misread the code. Sorry for that!